package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hsd",
-  "version": "4.99.0",
+  "version": "4.00.0-rc.1",
```

Comment:
does this middle number need to be single `0`? I dont care just wondering

Reply:
yep, it should. typo